### DISCUSSION TITLE
Need to update latest MCUx-SDK project on zephyr project

### DISCUSSION THREAD

**Discussion by vishnusudhan** [zephyrproject-rtos/zephyr#48623] [Ideas] [answered]
Hi,

I am using zephyr-rtos project version 3.1.99, for that I need to update the hal-nxp/MCUx-SDK for i.MXRT 1160 EVK. 
Currently its has 2.11.0 revision. But latest version is 2.12.0 .

Suggest how to update halnxp / MCUx-SDK driver to latest version ?is there any specific specific update command (west) available for that?

**Comment by dleach02** [comment, marked as answer]
@vishnusudhan, NXP will handle updating the Zephyr hal-nxp/MCUx-SDK to version 2.12.0. This is in process now.

**Reply by DerekSnell** [reply]
@vishnusudhan 
FYI, the MCUXpresso SDK has been updated to v2.12.0 in the hal_nxp, and here is the PR for the Zephyr repo to use this update
https://github.com/zephyrproject-rtos/zephyr/pull/48714

**Reply by vishnusudhan** [reply]
@DerekSnell  thanks for your update
